util/seed_db.js

```diff
@@ -0,0 +1,39 @@
+const Job = require("../models/Job");
+const User = require("../models/User");
+const faker = require("@faker-js/faker").fakerEN_US;
+const FactoryBot = require("factory-bot");
+require("dotenv").config();
+
+const testUserPassword = faker.internet.password();
+const factory = FactoryBot.factory;
+const factoryAdapter = new FactoryBot.MongooseAdapter();
+factory.setAdapter(factoryAdapter);
+factory.define("job", Job, {
+    company: () => faker.company.name(),
+    position: () => faker.person.jobTitle(),
+    status: () =>
+        ["interview", "declined", "pending"][Math.floor(3 * Math.random())], // random one of these
+});
+factory.define("user", User, {
+    name: () => faker.person.fullName(),
+    email: () => faker.internet.email(),
+    password: () => faker.internet.password(),
+});
+
+const seed_db = async () => {
+    let testUser = null;
+    try {
+        const mongoURL = process.env.MONGO_URI_TEST;
+        await Job.deleteMany({}); // deletes all job records
+        await User.deleteMany({}); // and all the users
+        testUser = await factory.create("user", { password: testUserPassword });
+        await factory.createMany("job", 20, { createdBy: testUser._id }); // put 30 job entries in the database.
```

Comment:
Hm 30 entries or 20? From here looks like 20 - wondering if the mismatch of 20 or 21 is because of the order of tests being run (if “create” is run first…)